board/piksiv3/rootfs-overlay/usr/bin/dump_core.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+/usr/bin/logger "CORE DUMP!! executable=$1 pid=$2 time=$3"
+echo "CORE DUMP!! executable=$1 pid=$2 time=$3" | /usr/bin/sbp_log --error
+
+corefile="/tmp/cores/core-$1-$2-$3"
+touch $corefile
+
+while read line
+do
```

Comment:
won't this mean we will fill up the ramfs completely rather than leaving some left over to allow the sytem to operate?